google/cloud/forseti/services/scanner/dao.py

```diff
@@ -59,7 +71,7 @@ class Violation(base):
         __tablename__ = violations_tablename
 
         id = Column(Integer, primary_key=True)
-        created_at = Column(DateTime())
+        created_at_timestamp = Column(String(256))
```

Comment:
This name change is much better and consistent with the public gcp documentations!  :+1: